statistics.py

```diff
@@ -1,5 +1,15 @@
-#!/usr/bin/env python
+#!/usr/bin/env runaiida
 # pylint: disable=import-outside-toplevel
+"""
+Script to get some anonymous statistics on the usage of the AiiDA profile.
+
+This mostly returns the type of nodes in the DB and their count.
+
+*NOTE*: You can just execute this script in your virtual environment
+if you want to get the results for your default AiiDA profile; otherwise, 
+run it with 
+`verdi -p PROFILE_NAME run statistics.py`. 
```

Comment:
```suggestion
*NOTE*: Simply execute ./statistics.py in your virtual environment to get the results for your 
default AiiDA profile; otherwise, run `verdi -p PROFILE_NAME run statistics.py`. 
```